docs/tempo/website/operations/server_side_metrics.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Server-side metrics
+weight: 12
+---
+
+# Server-side metrics
+
+<span style="background-color:#f3f973;">Server-side metrics is an experimental feature.</span>
+
+Server-side metrics is a feature that allows to derive metrics from ingested traces.
+
+To generate metrics, it uses an additional component: the metrics-generator.
+If present, the distributor will write received spans to both the ingester and the metrics-generator.
+The metrics-generator processes spans and writes metrics to a Prometheus datasource using the Prometheus remote write protocol.
+
+<!-- TODO: Expand section -->
```

Comment:
i think this is a fairly good intro. i'm not sure you need any more details for just the opener.

Reply:
Removed